FhirToDataLake/docs/Process FHIR extensions.md

```diff
@@ -0,0 +1,101 @@
+# Process FHIR extensions  
+
+ ![image](./assets/ProcessExtensions.png)
+
+## Generate parquet data for FHIR extensions
+
+### 1.	Prepare the customized schema templates
+For each resource types, e.g. Patient, Observation, you need to prepare a **liquid template** and a related **JSON schema** file.
+- The liquid template will be used to convert the raw FHIR JSON data into target structure.
+- The JSON schema will be used to generate the Parquet schema and create table definitions.
+
+Follow the links for more information about [liquid template](http://dotliquidmarkup.org/) and [JSON schema](https://json-schema.org/learn/getting-started-step-by-step).
+
+Example liquid template and JSON schema for “Patient” resource, flatten the “birthplace” extension for analytics.
```

Comment:
Below examples are intend for [Patient birthPlace](https://build.fhir.org/extension-patient-birthplace.html) extension, maybe add the reference for it.